### ISSUE TITLE
Cannot view entries

### ISSUE THREAD

**Issue by madrid1979** [CalderaWP/Caldera-Forms#2196]
## Do You Need Immediate Help?
YES

## Is This A Feature Request?
NO

## What Version Of Caldera Forms, WordPress and PHP Are You Using?
1.5.8.2

## Does Your Issue Persist When You Disable All Other Plugins and Switch To The Default Theme?
YES

## What Is The Unexpected Behaviour?
Viewing entries is no longer possible

## What PHP Errors Have You Logged While Reproducing This Bug?
`2018/01/04 00:22:10 [error] 5697#0: *238520 FastCGI sent in stderr: "PHP message: PHP Warning:  json_decode() expects parameter 1 to be string, array given in [...]/wp-content/plugins/caldera-forms/classes/admin.php on line 963`
(This warning repeats before...)
`2018/01/04 00:22:10 [error] 5697#0: *238520 upstream sent too big header while reading response header from upstream, client: 24.120.32.66, server: [my-server], request: "POST /wp-admin/admin-ajax.php HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "[my-host]", referrer: "[...]/wp-admin/admin.php?page=caldera-forms-pin-CF5a1f3662b8eb3"`

## What JavaScript Errors Have You Seen While Reproducing This Bug?
The top level error is:
`[Vue warn]: Cannot find element: #caldera-forms-clippy -- vue.min.js?ver=1.5.8.2:2`

Stack trace:

```
te @ vue.min.js?ver=1.5.8.2:2
Xa @ vue.min.js?ver=1.5.8.2:1
Ka.$mount @ vue.min.js?ver=1.5.8.2:3
Fa @ vue.min.js?ver=1.5.8.2:1
a._init @ vue.min.js?ver=1.5.8.2:2
Ka @ vue.min.js?ver=1.5.8.2:1
(anonymous) @ caldera-clippy.js?ver=1.5.8.2:45
(anonymous) @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:2
i @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:2
fireWith @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:2
(anonymous) @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:2
i @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:2
fireWith @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:2
y @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:4
c @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:4
XMLHttpRequest.send (async)
send @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:4
ajax @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:4
n.(anonymous function) @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:4
get @ caldera-clippy.js?ver=1.5.8.2:80
CalderaFormsAdminClippys2.init @ caldera-clippy.js?ver=1.5.8.2:35
(anonymous) @ admin.min.js?ver=1.5.8.2:1
i @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:2
fireWith @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:2
ready @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:2
K @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-widget,jquery-ui-position,utils,wp-pointer,zxcvbn-async,password-strength-meter,moxiejs,plu&load[]=pload&ver=4.9.1:2
```

**Comment by Shelob9** [comment]
@madrid1979 Thanks for using Caldera Forms. It looks like the root cause of this is the same as #1996 that will get fixed in 1.6.0 at the latest.

Please note Github issue tracker is a development tool for the open source plugin. We offer priority support and can help you implement a fix or work-around for your site via that channel. https://calderaforms.com/support

Closing this as a duplicate of #1996

**Comment by madrid1979** [comment]
Awesome, thank you!
